crates/cli/tests/snapshots/codegen__codegen_rust.snap

```diff
@@ -693,6 +690,10 @@ impl RepeatingTestArg {
 	#[allow(unused)]
 	pub fn find_by_scheduled_id(scheduled_id: u64) -> Option<Self> {
 		Self::find(|row| row.scheduled_id == scheduled_id)
+}
+	#[allow(unused)]
+	pub fn filter_by_scheduled_at(scheduled_at: ScheduleAt) -> TableIter<Self> {
```

Comment:
This got added in Rust output, but not other langs, creating inconsistency between language APIs.

Let's exclude it for now - if we want to allow more types in addition to what we decided in the consistent filtering proposal, we should do this (ugh can't avoid repeating that word) consistently.

Reply:
Yeah, I was just updating everywhere `is_special` was used but it looks like the logic is structured a lil differently across generators. Will revert.